### PR TITLE
Add capistrano

### LINKED ISSUE #116: Setup app for capistrano/deploybot for staging and production
### Descriptive summary

It's been decided we'll use the existing library staging and production VMs to deploy this app, primarily to keep things simple for getting the app out the door.

TODO:

- [ ] Get the Rails master key in place in the env variables repo
- [ ] [Get the DB info put into place for the app for staging and prod](#108)
- [ ] Get deploybot updated with hifive config needed
- [ ] Get capistrano setup in hifive to deploy to staging/prod
- [ ] Get passenger/ruby/rails etc. entries setup in staging and prod
...

### Rationale

Deploy deploy deploy.

### Expected behavior
We can use the H5 app in the staging server, authenticate w/ shib, create records, send emails, etc.

### Actual behavior
Um, i mean we can use docker on our laptops..

### Related work
#108 

Link to related tickets or prior related work here.

### PR DESCRIPTION
**Pull request by VivianChu**
Fixes #116 , #127 , #130, #131 , #129 

#### Local Checklist
- [ ] Tests written and passing locally?
- [ ] Code style checked?
- [ ] QA-ed locally?
- [x] Rebased with `master` branch?
- [x] Configuration updated (if needed)?
- [ ] Documentation updated (if needed)?

#### What does this PR do?
Add capistrano setup in hifive to deploy to staging/prod.
Fix the Recognizer is not getting stored/persisted issue

#### Manual testing steps?
https://lib-hifive-staging.ucsd.edu/hifive/

#### Screen shots
hifiveemail

hifiveemailfixurl

#### New ENV variables
References https://github.com/ucsdlib/env-variables/issues/37

@mcritchlow - please review

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ ruby '2.5.3'
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '~> 5.2.2'
 # Use postgresql as the database for Active Record
-gem 'pg', '>= 0.18', '< 2.0'
+gem 'pg', '1.1.3'
```
Comment (mcritchlow) [inline Gemfile]: Do we need to lock to this version or can we be more lenient? I'm curious where this one is coming from. The downside to locking like this, I think, is I'm not sure whether dependabot will make PRs or not..

Comment (VivianChu) [reply Gemfile]: @mcritchlow - I revert the pg gem change and the deploy still works.

**Comment by VivianChu** [comment]
@mcritchlow - This PR also includes the fix for  Recognizer is not getting stored/persisted issue.  It works fine in staging.  Above is the screen shot.

**Comment by VivianChu** [comment]
@mcritchlow - This PR also includes the fix for the url helper issue in the email.  I added the screen shot to the description above.

**Pull request merged**